Makefile

```diff
@@ -37,4 +37,34 @@ depends:
 	go get -d ./...
 
 build: depends
-	cd vflow; go build $(LDFLAGS) -o vflow $(GOFILES)
+	cd vflow; go build $(LDFLAGS) 
+
+build-windows: depends
+	cd vflow; gox $(LDFLAGS) 
```

Comment:
GOX is cool but why you think we need to have it at vFlow and add one more 3rd party package?